glustercli/cmd/utils.go

```diff
@@ -66,33 +66,29 @@ func sizeToBytes(value string) (uint64, error) {
 	return size, nil
 }
 
-// logn is used to find the unit size the given MB belongs to.
-// 1024 MB will return 1
-// 1048576 MB will return 2 and so on
+// logn is used to find the unit size the given bytes belongs to.
+// 1024 will return 1
+// 1048576 will return 2 and so on
 func logn(n, b float64) float64 {
 	return math.Log(n) / math.Log(b)
 }
 
-// humanReadable converts size given in MB into a more human readable unit
+// humanReadable converts size given in bytes into a more human readable unit
 //
-// humanReadable(1024) returns 1.0 GB
-// humanReadable(1536) returns 1.5 GB
-// humanReadable(1048576) returns 1.0 TB
+// humanReadable(1024) returns 1.0 KiB
+// humanReadable(1536) returns 1.5 KiB
+// humanReadable(1048576) returns 1.0 MiB
 func humanReadable(value uint64) string {
-	units := []string{"MB", "GB", "TB", "PB", "EB"}
-	// If less than 1024MB we return it as such
+	units := []string{"B", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB"}
+
+	// If less than 1024 we return it as such
 	if value < 1024 {
-		return fmt.Sprintf("%.1f MB", float64(value))
+		return fmt.Sprintf("%.1f", float64(value))
```

Comment:
Sure, I will add `B` suffix to it. Old format of `MB` is not exposed to users.

Also in our case we will not hit this since minimum volume size is `20M`, Bytes value printed for the sake of this library function completeness.

Reply:
updated now